components/konflux-ui/staging/base/proxy/proxy.yaml

```diff
@@ -66,13 +113,21 @@ spec:
 
             chmod 640 /mnt/nginx-generated-config/auth.conf
 
+            # Generate watson-auth.conf with Basic auth from Watson API key
+            watson_basic=$(cat /mnt/watson-api-key/API_KEY)
```

Comment:
Add the base64 encoding and append it to `apikey: <base64-token>` that will be your watson_basic.